### ISSUE TITLE
shapeshifter-dispatcher fails to compile: addInstance.Bytes undefined

### ISSUE THREAD

**Issue by meskio** [OperatorFoundation/shapeshifter-dispatcher#27]
It seems that the master of the repo is failing to compile:
```
shapeshifter-dispatcher/shapeshifter-dispatcher❯ go build
# github.com/OperatorFoundation/shapeshifter-transports/transports/Replicant/toneburst
../../shapeshifter-transports/transports/Replicant/toneburst/monotone.go:34:26: addInstance.Bytes undefined (type monolith.Instance has no field or method Bytes)
../../shapeshifter-transports/transports/Replicant/toneburst/monotone.go:71:43: addInstance.Bytes undefined (type monolith.Instance has no field or method Bytes)
```

Is it possible that this `Bytes` function was not commited?

**Comment by meskio** [comment]
The problem seems to be https://github.com/OperatorFoundation/monolith-go/commit/b4f585cf0062cb70ad3f5bf57571733874063ec4#diff-22eef1e1d10b43ba66bb778f9b8c01f4L11

It looks like `Bytes()` was deleted in this commit, not sure if replaced by `Messages()`.

**Comment by consuelita** [comment]
This has been resolved with release 2.1.1.  Thank you!
